### PR TITLE
Update teams-app-setup-policies.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8436
Added note

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-app-setup-policies.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ As an admin, you can use app setup policies to install and pin apps to promote t
 
 ## Pin apps
 
+> [!NOTE]
+> Pinning an app via setup policy will also install it.
```
Comment (JohanFreelancer9) [inline Teams/teams-app-setup-policies.md:35]:
```suggestion
> Pinning an app via an app setup policy will also install it.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

File: Teams/teams-app-setup-policies.md [hunk 2]
```diff
@@ -31,6 +31,9 @@ As an admin, you can use app setup policies to install and pin apps to promote t
 
 ## Pin apps
 
+> [!NOTE]
+> Pinning an app via an app setup policy will also install it.
```
Comment (JohanFreelancer9) [inline Teams/teams-app-setup-policies.md:35]:
```suggestion
> Pinning an app via an app setup policy will also install the app.
```

Comment (JohanFreelancer9) [inline Teams/teams-app-setup-policies.md:35]:
```suggestion
> Pinning an app via an app setup policy will also install the app.
```

**Comment by JohanFreelancer9** [comment]
@lindspea, apologies, kindly commit the additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ashishguptaiitb** [comment]
Let's close this PR @lindspea . I've updated the article (at https://docs.microsoft.com/microsoftteams/teams-app-setup-policies) via another PR.

**Pull request closed without merging**